packages/wrangler/bin/wrangler.js

```diff
@@ -87,9 +87,6 @@ function runDelegatedWrangler() {
 	} = JSON.parse(fs.readFileSync(packageJsonPath));
 	const resolvedBinaryPath = path.resolve(packageJsonPath, "..", binaryPath);
 
-	console.log(
```

Comment:
Actually I think this message to the user is really useful. I think we should keep it but move it to `logger.log()` so that it can be filtered out.

Reply:
Ok, makes sense to me.